src/kenigsberg/MorseCode/MorseCode.java

```diff
@@ -0,0 +1,129 @@
+package kenigsberg.MorseCode;
+
+public class MorseCode
+{
+
+    final String[] morseCode;
+    final String[] letters;
+
+    public MorseCode()
+    {
+
+        this.morseCode = new String[26]; //every object created, creates an array of morse code values each index matching to letter in alphabet
```

Comment:
Create the array without having to set every single item in the array.
https://stackoverflow.com/questions/1938101/how-to-initialize-an-array-in-java